### PR TITLE
style: colums and column classes to tailwind

### LINKED ISSUE #9191: replace `columns` & `column` with tailwind equivalent
ref #9013

### PR DESCRIPTION
**Pull request by iMac7**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [ ] Closes #9191 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [ ] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by prury** [comment]
hey @iMac7 I'm going to to throw here what i've found halfway trough: 
make sure to compare to canary and check mobile when replacing styles with tailwind, ty

mobile: 
[image](https://github.com/kodadot/nft-gallery/assets/36627808/a3f3dd45-a7a8-4c04-adef-4856a1997b46)

mobile: 
[image](https://github.com/kodadot/nft-gallery/assets/36627808/355e672c-86c0-4cd6-839e-44aeecd62835)

mobile: 
[image](https://github.com/kodadot/nft-gallery/assets/36627808/f263e0db-f8a3-4370-b62b-eb370de0ef1a)

too close: 
[image](https://github.com/kodadot/nft-gallery/assets/36627808/0f437884-471a-4be7-885d-47eaae92d2de)

desktop:
https://github.com/kodadot/nft-gallery/assets/36627808/9157548e-b43e-4b78-9dac-7dd42d88cbff

desktop:
https://github.com/kodadot/nft-gallery/assets/36627808/1fecf527-b25a-47b9-84b3-835584ed3730

desktop:
https://github.com/kodadot/nft-gallery/assets/36627808/b23049c9-bbfe-4af4-9f54-91ad86e778ef

desktop:
https://github.com/kodadot/nft-gallery/assets/36627808/fac9c3db-b3e1-4029-af12-26f4edbe8375

desktop:
[image](https://github.com/kodadot/nft-gallery/assets/36627808/c8361343-b2fe-433d-8112-3600e138fb08)

desktop:
[image](https://github.com/kodadot/nft-gallery/assets/36627808/28dc6519-b882-4049-9424-9e183fdd414a)

**Comment by iMac7** [comment]
> hey @iMac7 I'm going to to throw here what i've found halfway trough: make sure to compare to canary and check mobile when replacing styles with tailwind, ty
> 
> mobile: [image](https://github.com/kodadot/nft-gallery/assets/36627808/a3f3dd45-a7a8-4c04-adef-4856a1997b46)
> 
> mobile: [image](https://github.com/kodadot/nft-gallery/assets/36627808/355e672c-86c0-4cd6-839e-44aeecd62835)
> 
> mobile: [image](https://github.com/kodadot/nft-gallery/assets/36627808/f263e0db-f8a3-4370-b62b-eb370de0ef1a)
> 
> too close: [image](https://github.com/kodadot/nft-gallery/assets/36627808/0f437884-471a-4be7-885d-47eaae92d2de)
> 
> desktop: https://github.com/kodadot/nft-gallery/assets/36627808/9157548e-b43e-4b78-9dac-7dd42d88cbff
> 
> desktop: https://github.com/kodadot/nft-gallery/assets/36627808/1fecf527-b25a-47b9-84b3-835584ed3730
> 
> desktop: https://github.com/kodadot/nft-gallery/assets/36627808/b23049c9-bbfe-4af4-9f54-91ad86e778ef
> 
> desktop: https://github.com/kodadot/nft-gallery/assets/36627808/fac9c3db-b3e1-4029-af12-26f4edbe8375
> 
> desktop: [image](https://github.com/kodadot/nft-gallery/assets/36627808/c8361343-b2fe-433d-8112-3600e138fb08)
> 
> desktop: [image](https://github.com/kodadot/nft-gallery/assets/36627808/28dc6519-b882-4049-9424-9e183fdd414a)

wow quickly switching tabs is a great way to test especially for me working with just my laptop XD thanks for the tip

**Comment by iMac7** [comment]
That last one (`OverviewTable`) requires that I have at least one row of data in the table to work with. I've not figured out how to buy or mint an nft or anything that requires money on localhost (If you know how to get test dot or ksm please share, the docs left me hanging at that point, no response on telegram yet). 

Tiny differences in the Responsivetable on desktop (apparently each cell had its own padding before?). Workable but lmk if changes are needed there.

Sorry if many errors from me this needs time, coffee and the eye of a designer to get right :grinning:

**Comment by prury** [comment]
> That last one (`OverviewTable`) requires that I have at least one row of data in the table to work with. I've not figured out how to buy or mint an nft or anything that requires money on localhost (If you know how to get test dot or ksm please share, the docs left me hanging at that point, no response on telegram yet).
> 
we lack a testnet at the moment sir, i'll test it

> 
> Sorry if many errors from me this needs time, coffee and the eye of a designer to get right 😀

haha, thank you for your work, and yup, we know, lots of small details to check

File: components/gallery/GalleryItem.vue
Comment (prury) [inline components/gallery/GalleryItem.vue:1]: still need some adjustments

File: components/massmint/OverviewTable.vue
Comment (prury) [inline components/massmint/OverviewTable.vue:1]: massmint: 

image

File: components/migrate/steps/Index.vue
Comment (prury) [inline components/migrate/steps/Index.vue:1]: migrate steps should be centered as canary:

image

File: components/profile/ProfileDetail.vue
Comment (prury) [inline components/profile/ProfileDetail.vue:1]: lacking small gap on profile: 
https://github.com/kodadot/nft-gallery/assets/36627808/6335992b-cb5d-4702-bc94-fcb8416a62e1

File: components/profile/activityTab/History.vue
Comment (prury) [inline components/profile/activityTab/History.vue:1]: text truncation was lost:
image

item spacing(too close):
https://github.com/kodadot/nft-gallery/assets/36627808/b3fa5ce6-3f1c-4dbe-88c4-45195aebda2d

File: components/shared/form/RadioSelect.vue
Comment (prury) [inline components/shared/form/RadioSelect.vue:1]: radio on ahk->create->waifu
https://github.com/kodadot/nft-gallery/assets/36627808/7525a32b-7f45-44c1-8cc3-db038b71160f

**Comment by prury** [comment]
made some comments but we are almost there!

File: components/migrate/steps/Index.vue
Comment (prury) [reply components/migrate/steps/Index.vue:1]: still happening as well

File: components/profile/activityTab/History.vue
Comment (prury) [reply components/profile/activityTab/History.vue:1]: truncation ok but space between rows still the same

File: components/migrate/steps/Index.vue
Comment (iMac7) [reply components/migrate/steps/Index.vue:1]: I can't get to this part without an item to migrate :disappointed:

File: components/profile/activityTab/History.vue
Comment (iMac7) [reply components/profile/activityTab/History.vue:1]: Can't reach this as well I think, I have nothing on my profile yet :cry:

Comment (prury) [reply components/profile/activityTab/History.vue:1]: you can check my profile or anyone else:
https://canary.kodadot.xyz/ahk/u/CmWHiv7h4m9tEzKD94DH4mqwGTvsdYQe2nouWPF7ipmHpqA

File: components/migrate/steps/Index.vue
Comment (prury) [reply components/migrate/steps/Index.vue:1]: i think devs usually mock it, but i don't know how

**Comment by iMac7** [comment]
@prury for the tiny changes left that can't be easily mocked on the dev environment, maybe to be fixed later? how does this look like to merge so far?

**Comment by prury** [comment]
> @prury for the tiny changes left that can't be easily mocked on the dev environment, maybe to be fixed later? how does this look like to merge so far?

I think the best path is to undo tailwind modifications on the places you could not access and leave them for a follow up issue

**Comment by iMac7** [comment]
> I think the best path is to undo tailwind modifications on the places you could not access and leave them for a follow up issue

done

File: components/massmint/OverviewTable.vue
Comment (prury) [reply components/massmint/OverviewTable.vue:1]: mb on this one, is a canary problem with non-squared images

**Review by preschian — APPROVED** [review]

**Comment by yangwao** [comment]
@iMac7 please resolve conflicts

**Comment by yangwao** [comment]
image

**Comment by yangwao** [comment]
pay 50 usd
Hopefully won't break things

**Pull request merged**